cmd/mistry/main.go

```diff
@@ -187,40 +212,24 @@ EXAMPLES:
 					}
 				}
 
-				jr := types.JobRequest{Project: project, Group: group, Params: params}
-				jrJSON, err := json.Marshal(jr)
-				if err != nil {
-					return err
+				url := fmt.Sprintf("http://%s:%s/%s", host, port, JobsPath)
+				if async {
+					url += "?async"
 				}
 
-				request, err := http.NewRequest("POST", fmt.Sprintf("http://%s:%s/%s", host, port, JobsPath), bytes.NewBuffer(jrJSON))
+				body, err := sendRequest(url, verbose, project, group, params)
```

Comment:
Maybe better construct the `JobRequest` before calling this function and just pass it as a parameter? The function would then would only do what it's supposed to.